test/oci_artifacts.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+# vim:set ft=bash :
+
+load helpers
+
+function setup() {
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+ARTIFACT_IMAGE=quay.io/crio/seccomp:v2
```

Comment:
maybe should be added to IMAGES() in test/helpers.bash so we cache the image before running?

Reply:
Actually, the copyimg binary does not work with artifacts. Should we follow-up on that or is it fine as-is?

Reply:
Let's get this in. We can do this as a follow-up. This PR is already big.

Reply:
Sounds good, created https://github.com/cri-o/cri-o/issues/9041 for that.